cvat/settings/base.py

```diff
@@ -353,6 +353,20 @@ class CVAT_QUEUES(Enum):
         'func': 'cvat.apps.iam.utils.clean_up_sessions',
         'cron_string': '0 0 * * *',
     },
+    *(
+        {
+            'queue': CVAT_QUEUES.CLEANING.value,
+            'id': f'cron_{model.lower()}_export_cache_cleanup',
+            'func': 'cvat.apps.engine.cron.cron_export_cache_cleanup',
+            # Run once a day at midnight
```

Comment:
This is only applicable to projects. You've configured the other jobs to run at 6:00 and noon.

That said, what's the purpose of staggering the jobs like that? Why not just have one job that runs at midnight and cleans up everything?

Reply:
> This is only applicable to projects. You've configured the other jobs to run at 6:00 and noon.

That's because previously 3 jobs ran at midnight and I forgot to delete the comment.

> That said, what's the purpose of staggering the jobs like that? Why not just have one job that runs at midnight 
and cleans up everything?

I thought it would be better to split the "long" potential job into several to avoid clogging the queue since there are other jobs that execute in this queue.

Reply:
Do you think it would take a long time to run all three? I'd assume it would be pretty quick, since it's just deleting some files.

Reply:
> Do you think it would take a long time to run all three? 

I'll check with the real amount of data

Reply:
Okay, I’ve checked it on a queryset of 15k items, and it takes about 10 seconds. I guess that the actual numbers will be around 100–300k for tasks, less than 100k for projects, and well over 300k for jobs. As a result, this cron job could take significant time when a large number of tasks with many associated jobs are created within the last 30 days. This problem remains the same if there are 3 separate cron jobs. Based on this, I decided to merge separate jobs into one.

Reply:
We could probably select tasks/projects/jobs that have been exported within the last 2 weeks instead of 1 month

Reply:
How about limiting 1 function run just by a fixed number of oldest entries? The next run will continue from where the previous one finished,

Reply:
Here's a crazy idea. Instead of mucking around with last export dates, can we change the file layout so that all exports are saved in a dedicated directory? That way, the cleanup job would just have to iterate over that directory. This way, it'll be impossible to miss any stale exports.

Reply:
> I guess that the actual numbers will be around 100–300k for tasks, less than 100k for projects, and well over 300k for jobs.

FWIW, I'm pretty sure we have analytics events for exports, so you could see what the actual numbers are (although there might not be events for backups).

> As a result, this cron job could take significant time when a large number of tasks with many associated jobs are created within the last 30 days. 

I might be misunderstanding something, but why does it matter how many tasks/jobs are _created_? I thought this was about exports.